src/main/java/org/interledger/core/InterledgerAddress.java

```diff
@@ -0,0 +1,70 @@
+package org.interledger.core;
```

Comment:
My rationale for the package naming was that I see `org.interledger` as the project prefix. There are many aspects to "Interledger" including the ILP protocol but also other protocols like SPSP.

So stuff that is core ILP is in the `org.interledger.ilp` namespace.